src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/ApprenticeshipResumedEventHandler.cs

```diff
@@ -1,47 +1,51 @@
+using SFA.DAS.CommitmentsV2.Data.Extensions;
+using SFA.DAS.CommitmentsV2.Models;
 
 namespace SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers
 {
     public class ApprenticeshipResumedEventHandler : IHandleMessages<ApprenticeshipResumedEvent>
     {
-        private readonly IMediator _mediator;
+        private readonly Lazy<ProviderCommitmentsDbContext> _dbContext;
+        private readonly IEncodingService _encodingService;
         private readonly ILogger<ApprenticeshipResumedEventHandler> _logger;
 
-        public ApprenticeshipResumedEventHandler(IMediator mediator, ILogger<ApprenticeshipResumedEventHandler> logger)
+        public ApprenticeshipResumedEventHandler(Lazy<ProviderCommitmentsDbContext> dbContext, IEncodingService encodingService, ILogger<ApprenticeshipResumedEventHandler> logger)
         {
-            _mediator = mediator;
+            _dbContext = dbContext;
             _logger = logger;
+            _encodingService = encodingService;
         }
 
         public async Task Handle(ApprenticeshipResumedEvent message, IMessageHandlerContext context)
         {
             _logger.LogInformation($"Received {nameof(ApprenticeshipResumedEventHandler)} for apprentice {message?.ApprenticeshipId}");
 
-            var apprenticeshipQueryResult = await _mediator.Send(new GetApprenticeshipQuery(message.ApprenticeshipId));
+            var apprenticeship = await _dbContext.Value.GetApprenticeshipAggregate(message.ApprenticeshipId, default);
 
-            var emailToProviderCommand = BuildEmailToProviderCommand(apprenticeshipQueryResult, message.ResumedOn);
+            var emailToProviderCommand = BuildEmailToProviderCommand(apprenticeship, message.ResumedOn);
 
             await context.Send(emailToProviderCommand, new SendOptions());
         }
 
-        private SendEmailToProviderCommand BuildEmailToProviderCommand(GetApprenticeshipQueryResult apprenticeship, DateTime resumeDate)
+        private SendEmailToProviderCommand BuildEmailToProviderCommand(Apprenticeship apprenticeship, DateTime resumeDate)
         {
-            var sendEmailToProviderCommand = new SendEmailToProviderCommand(apprenticeship.ProviderId,
+            var sendEmailToProviderCommand = new SendEmailToProviderCommand(apprenticeship.Cohort.ProviderId,
                 "ProviderApprenticeshipResumeNotification",
                       new Dictionary<string, string>
                       {
-                                  {"EMPLOYER", apprenticeship.EmployerName},
-                                  {"APPRENTICE", apprenticeship.FirstName},
+                                  {"EMPLOYER", apprenticeship.Cohort.AccountLegalEntity.Name},
+                                  {"APPRENTICE",  $"{apprenticeship.FirstName} {apprenticeship.LastName}"},
                                   {"DATE", resumeDate.ToString("dd/MM/yyyy")},
-                                  {"URL", $"{apprenticeship.ProviderId}/apprentices/manage/Hashed/details"}
```

Comment:
how did this url ever work 😱

Reply:
it didn't, was missed in the initial check in.